libs/galley-types/src/Galley/Types/Teams.hs

```diff
@@ -261,7 +261,7 @@ instance FromJSON FeatureFlags where
       <*> (fromMaybe (Defaults defaultAppLockStatus) <$> (obj .:? "appLock"))
       <*> (fromMaybe defaultClassifiedDomains <$> (obj .:? "classifiedDomains"))
       <*> (fromMaybe (Defaults (TeamFeatureStatusNoConfig TeamFeatureEnabled)) <$> (obj .:? "fileSharing"))
-      <*> (fromMaybe (Defaults (TeamFeatureStatusNoConfig TeamFeatureEnabled)) <$> (obj .:? "conferenceCalling"))
+      <*> (fromMaybe (Defaults defaultConferenceCalling) <$> (obj .:? "conferenceCalling"))
```

Comment:
I think this feature needs a default config that depends on the lock status.  It is always either locked/disabled or unlocked/enabled.  No further changes are possible.  This can be accomplished with something like:

```haskell
class HasDefaultConfig (a :: TeamFeatureName) where
  defaultConfig :: LockStatusValue -> a
```

It can then be either constant and ignore the argument (like in self deleting messages), or consider it (like here).